networks/filecoin/rpc.csv

```diff
@@ -1,26 +1,26 @@
 slug,provider,plan,nodeType,chain,accessPrice,queryPrice,uptimeSla,bandwidthSla,blocksBehindSla,starred,trial,availableApis,limitations,securityImprovements,monitoringAndAnalytics,regions,verifiedUptime,verifiedLatency,verifiedBlocksBehindAvg,actionButtons,address
-ankr-mainnet-enterprise,!provider:ankr-enterprise-recent-state,,Recent-State,mainnet,,,,,,,,"[""All read methods + MPoolPush"", ""Advanced Ankr API""]","[""1500 requests per second""]","[""DDoS Protection"", ""IP Protection"", ""Domain name protection"", ""Custom rate limiting"", ""Error tracing""]","[""per-ip analytics"", ""per-country analytics""]",,,,,"[""[Buy](https://www.ankr.com/rpc/home/)"", ""[Docs](https://www.ankr.com/docs/rpc-service/chains/chains-list/#filecoin)""]",
-ankr-mainnet-free,!provider:ankr-free-recent-state,,Recent-State,mainnet,,,,,,,,"[""All read methods + MPoolPush""]","[""1 key only""]","[""DDoS Protection"", ""IP Protection"", ""Domain name protection"", ""Custom rate limiting"", ""Method Limiting"", ""Error tracing"", ""Contract addresses protection""]",null,,,,,"[""[Buy](https://www.ankr.com/rpc/home/)"", ""[Docs](https://www.ankr.com/docs/rpc-service/chains/chains-list/#filecoin)""]",
-ankr-mainnet-pay-as-you-go,!provider:ankr-pay-as-you-go-recent-state,,Recent-State,mainnet,,,,,,,,"[""All read methods + MPoolPush"", ""Advanced Ankr API""]","[""1500 requests per second""]","[""DDoS Protection"", ""IP Protection"", ""Domain name protection"", ""Custom rate limiting"", ""Error tracing""]","[""per-method analytics"", ""latency"", ""CU analytics""]",,,,,"[""[Buy](https://www.ankr.com/rpc/home/)"", ""[Docs](https://www.ankr.com/docs/rpc-service/chains/chains-list/#filecoin)""]",
-chain-love-calibnet-free,!provider:chain-love-free,,Recent-State,calibnet,$0,$0,,,,,,"[""All read methods + MPoolPush""]","[""100 requests per minute""]",null,null,"[""Asia""]",,,,"[""[Send requests](https://calibration.filecoin.chain.love/rpc/v0)"", ""[Docs](https://protofire-org.gitbook.io/proteus-shield-docs/blockchains/filecoin)""]",
```

Comment:
Read methods and pricing was left intentionally here, because chain-love provider is having them DIFFERENT network to network